bricks/template/__brick__/{{project_name.snakeCase()}}/lib/storage/secure_storage.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:injectable/injectable.dart';
+
+const _keyAccessToken = 'KEY_ACCESS_TOKEN';
+
+abstract class SecureStorage {
+  Future<String?> get accessToken;
+
+  Future<void> storeAccessToken(String accessToken);
+
+  Future<void> clearAllStorage();
```

Comment:
`clearAll` is enough IMO

```suggestion
  Future<void> clearAll();
```